{"frames":[{"character_number":0,"end_character_number":0,"end_line_number":0,"file":"","func":"load(\"./F03_SelPredictores/F03_4_ga_100.RData\")","line_number":0},{"character_number":0,"end_character_number":0,"end_line_number":0,"file":"","func":"readChar(con, 5L, useBytes = TRUE)","line_number":0}],"message":"Error in readChar(con, 5L, useBytes = TRUE) : cannot open the connection\n"}